adding_species_manuscript.tex

```diff
@@ -660,9 +660,8 @@ \subsection*{Filling in the missing pieces}
 The table also mentions possible consequences of misspecification of each parameter.
 
 
-There are additional considerations that apply to species with partial genome builds.
-We discussed above our choice in \stdpopsim to focus on species with chromosome-level assemblies.
-However, simulation is still potentially useful for species with partial genome builds.
+In some cases, one may wish to generate simulations for a species with a partial genome build.
+Despite the focus of \stdpopsim  on species with chromosome-level assemblies (see discussion above), simulation is still potentially useful for species with less complete assemblies, with some important considerations to keep in mind.
 Longer contigs or scaffolds in these builds can be simulated separately and independently.
```

Comment:
@bhaller What do you think of this? The 2nd sentence is a bot long, but I think it works.

Reply:
Sure, works for me.  :->